db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160217010342) do
+ActiveRecord::Schema.define(version: 20160228210723) do
```

Comment:
This is the source of the merge conflict.  You can see this if you try to merge master into your branch, or try to rebase your branch onto master:

<img width="589" alt="screen shot 2016-02-29 at 12 24 24 pm" src="https://cloud.githubusercontent.com/assets/1056957/13403038/092832ae-dee1-11e5-9676-792e7d92e8c5.png">

This is happening because other migrations were merged into master after the point you cut this branch.  So `master` has one version number for the schema.  And your code created its own version number when you ran it locally.  So this is saying "hey, maybe there are upstream changes that your migration hasn't considered."  This is usually not an actual conflict, but the warning here is a helpful reminder and easy to fix.

We can do this tomorrow in person, but you can typically resolve this by looking at other recent migrations, verifying there's nothing conflicting, and then updating the `schema.rb` file to take the more recent version number.  More background is [here](http://stackoverflow.com/questions/2979059/is-the-version-number-in-rails-schema-rb-used-for-anything/4047884#4047884).